### ISSUE TITLE
Keyring window appears on every connect/disconnect

### ISSUE THREAD

**Issue by CS-JS** [FreeRDP/Remmina#1149]
We support Remmina 1.2+ versions only.
Please do not report bugs occurring on older versions!

Please give us some details about the systems you are using:
* Client (OS name and version): Ubuntu 16.04.2
* Remmina version (remmina --version): remmina - Version 1.2.0-rcgit-18 (git rcgit-18)
* Desktop environment (GNOME, Unity, KDE, ..): KDE
* Connecting to (OS and version): Win7/8/10, WinServer2003-2016
* Connecting via (RDP, VNC, ...): RDP
* RDP issues only, freerdp version (xfreerdp --version): This is FreeRDP version 2.0.0~dev (git n/a)

Hi,

I have a strange remmina behaviour, when I connect to windows clients and servers via RDP. Everytime I connect to a machine, I get asked to unlock my keyring. Strange thing is, I don't have any keyring-features installed or enabled (that I am aware of), and also don't want to save my passwords.
Next strange thing is, the same window appears, when I disconnect from any client, or my session gets terminated (e.g. someone else connects to the same machine and steals my session ;-)).

Is there any option that I am currently not aware of, to turn these windows off?

![image](https://cloud.githubusercontent.com/assets/17683794/24516689/d6f2c2e0-157b-11e7-991b-eae829ee6426.png)
The screenshot reads german for "Please enter password to unlock keyring: Your keyring has not been automatically unlocked when you logged in to your machine"

Thanks for any advice.

Output of dpkg -l | grep keyring:
```
jschoen@NB-CS-JS:~$ dpkg -l | grep keyring
rc  gnome-keyring                                   3.18.3-0ubuntu2                                                               amd64        GNOME keyring services (daemon and tools)
ii  libgnome-keyring-common                         3.12.0-1build1                                                                all          GNOME keyring services library - data files
ii  libgnome-keyring0:amd64                         3.12.0-1build1                                                                amd64        GNOME keyring services library
ii  libp11-kit-gnome-keyring:amd64                  3.18.3-0ubuntu2                                                               amd64        GNOME keyring module for the PKCS#11 module loading library
ii  libpam-gnome-keyring:amd64                      3.18.3-0ubuntu2                                                               amd64        PAM module to unlock the GNOME keyring upon login
ii  ubuntu-keyring                                  2012.05.19                                                                    all          GnuPG keys of the Ubuntu archive
jschoen@NB-CS-JS:~$
```

**Comment by giox069** [comment]
Yes, that's true. Keyring is unlocked at startup and maybe accessed even if you don't save passwords. Passwords and other sessions settings are usually saved at the end of the session, so it's ok to have a second message at the end of the session.
If you want a different behavior, do one of the following, please rename this issue as "Feature request: add a global option to disable both password saving and keyring access", and we will mark as new feature request.

**Comment by CS-JS** [comment]
Thanks for the reply - I've just renamed the issue. Its not that important, I'm just gonna hit ESC after every connect, but I wanted to know if this is a behaviour maybe I introduced by installing some keyring packets as dependencies to other tools. I remember some time (maybe pre rcgit-18, can't tell for sure), where this window was not present, so I thought maybe if a certain keyring package is installed, remmina  automatically tries to use it, even if it doesn't make much sense in my case.

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @CS-JS @giox069
